src/components/common/OptionPopper.tsx

```diff
@@ -33,51 +33,55 @@ const useLayout = makeStyles({
   },
 });
 
+export interface Option {
+  styles: any;
+  onClick: () => void;
+  name: string;
+}
+
 const OptionPopper = ({
-  id,
   open,
   anchorEl,
-  onEditClick,
-  onDeleteClick,
+  onClickAway,
+  options,
 }: {
-  id: string | undefined;
   open: boolean;
   anchorEl: HTMLButtonElement | null;
-  onEditClick: () => void;
-  onDeleteClick: () => void;
+  onClickAway: () => void;
+  options: Array<Option>;
 }) => {
-  const optionBtn = options();
   const classes = useLayout();
+  const optionStyle = useOptions();
 
   return (
     <Popper
```

Comment:
there still should be a key for these poppers - i would suggest using the `ccpId` as that's guaranteed to be unique per row
![image](https://user-images.githubusercontent.com/59516223/90712022-1e40d000-e270-11ea-9b47-f0338e24dad7.png)

Reply:
Sorry forgot to address that warning. I made all the changes now!